runtime/port/zos390/j9csrsi.c

```diff
@@ -61,6 +61,8 @@ j9csrsi_init(int32_t *ret)
 						 CSRSI_REQUEST_V2CPC_LPAR | CSRSI_REQUEST_V3CPC_VM);
 		sess->infoarea_len = sizeof(sess->infoarea);
 
+		/* defined in j9csrsi_wrp.s. */
+		extern void j9csrsi_wrp(struct j9csrsi_t *session);
```

Comment:
Declarations must appear at the beginning of a block, or at the global level. I suggest this declaration belongs in `j9csrsi.h`.

Reply:
Thanks, moved the declaration to the header and tested/confirmed that it works.